safedata_validator/zenodo.py

```diff
@@ -799,63 +800,43 @@ def dataset_description(
     gbif_taxon_index = metadata.get("gbif_taxa")
     ncbi_taxon_index = metadata.get("ncbi_taxa")
 
-    gbif_text = (
-        " If a dataset uses a synonym, the accepted usage is shown followed by the "
-        "dataset usage in brackets. Taxa that cannot be validated, including new "
-        "species and other unknown taxa, morphospecies, functional groups and "
-        "taxonomic levels not used in the GBIF backbone are shown in square brackets."
-        ""
-    )
-
-    ncbi_text = (
-        " If a dataset uses a synonym, the accepted usage is shown followed by the "
-        "dataset usage in brackets. Taxa that cannot be validated, e.g. new or unknown "
-        "species are shown in square brackets. Non-backbone taxonomic ranks (e.g. "
-        "strains or subphyla) can be validated using the NCBI database. However, they "
-        "will only be shown if the user explicitly provided a non-backbone taxon. When"
-        " they are shown they will be accompanied by an message stating their rank."
-    )
-
     # When NCBI is absent use the old format for backwards compatibility
-    if gbif_taxon_index and not ncbi_taxon_index:
+    if gbif_taxon_index or ncbi_taxon_index:
         desc += tags.p(
             tags.b("Taxonomic coverage: "),
             tags.br(),
-            f" All taxon names are validated against the GBIF backbone taxonomy."
-            f"{gbif_text}",
-            taxon_index_to_text(gbif_taxon_index, True),
-        )
-    elif gbif_taxon_index and ncbi_taxon_index:
-        desc += tags.p(
-            tags.b("Taxonomic coverage: "),
-            tags.br(),
-            " For this dataset taxon names are validated against both the GBIF backbone"
-            " taxonomy and the NCBI taxonomy database.",
+            "This dataset contains data associated with taxa and these have been "
```

Comment:
This approach is definitely simpler so I can see the advantage of it. The reason it was so convoluted before is that I was aiming to ensure that a GBIF dataset uploaded without NCBI would generate the same text with the new setup as it would have done using the old. If we don't actually care about have that kind of backwards compatibility then I agree this is a much better setup

Reply:
Yeah - I don't think we care about being able to reproduce the same text. This is just one possible layout of the same core metadata. Even if we did go through and systematically change all the Zenodo descriptions (we can do that - it is only the deposit files that can't change), the information would remain the same and only the layout would change.

Reply:
Yeah that makes sense